plugins/git-build.js

```diff
@@ -10,7 +10,7 @@ exports.init = function(config, cimpler) {
    paths = Array.isArray(paths) ? paths : [paths]
    paths.forEach(function(repoPath) {
       var consumer = buildConsumer(config, cimpler, repoPath);
-      cimpler.consumeBuild(consumer, config.repoRegex);
+      cimpler.consumeBuild(consumer, config.repoRegex || config.buildFilter);
```

Comment:
I sort of think that the regex should just be a function in the config that does the regex matching, that way we don't need functionality around the regex || function in the application.

 but I don't feel strongly about it, so don't feel obligated to change it

Reply:
Ya I kinda thought the same thing but I didn't want to break functionality for existing configs (although updating them to wrap the regex wouldn't be the worst thing).

Reply:
Maybe let's make it deprecated? (remove it from the docs, but still support it)